### DISCUSSION TITLE
Wiring for ESP32CAM

### DISCUSSION THREAD

**Discussion by Eisenhauer1987** [tbnobody/OpenDTU#1740] [Q&A]
Hi, I have a ESP32CAM Module here. It is possible to wire an NRF24 to this Module. Can somebody help me with the right gpio pins to use?

**Comment by Omega13x** [comment]
Ich glaube du suchst das hier. 

[ESP32-Cam_nrf24L01+_als_DTU.pdf](https://github.com/tbnobody/OpenDTU/files/14259930/ESP32-Cam_nrf24L01%2B_als_DTU.pdf)
![Verdrahtung_ESP32-Cam_NRF24L01+](https://github.com/tbnobody/OpenDTU/assets/123419851/1c73239e-6041-4fa6-9505-ea80b3cf212f)
[ESP32-Cam_pinmapping.json](https://github.com/tbnobody/OpenDTU/files/14259988/ESP32-Cam_pinmapping.json)

Diese Firmware.: [https://github.com/tbnobody/OpenDTU/releases/download/v24.2.12/opendtu-generic_esp32.factory.bin](url)

**Reply by Eisenhauer1987** [reply]
Danke, ich probier das später!

Unfourtunaly I cannot add my herf 1800 because the serial number have a different layout than expected. I there any change to add this Hoymiles sub brand to a opendtu?

**Reply by Birkenstab** [reply]
Schau mal hier. Eventuell kannst du das auch auf den herf 1800 übertragen: https://github.com/tbnobody/OpenDTU/issues/1751

**Reply by Eisenhauer1987** [reply]
Perfekt, das geht, es werden zwar nur 2 String erkannt, aber ein erster schritt. Ich mach jetzt in #1751 weiter

**Reply by xdatravelbug** [reply]
Pinmapping works like a charm on ESP32-CAM (here for Herf-800). I tried before by using GPIOs 2 and 4 instead of 0 and 16 and also had different assignments, but that was not working for whatever reason. Maybe a contact issue...!?
If you want to have the LEDs as well use 33 for LED0 and 4 for LED1.
